src/main/java/com/hubspot/jinjava/interpret/Context.java

```diff
@@ -526,4 +527,12 @@ public void addDependencies(SetMultimap<String, String> dependencies) {
   public SetMultimap<String, String> getDependencies() {
     return this.dependencies;
   }
+
+  public boolean isHideInterpreterErrors() {
```

Comment:
even though this is a boolean, it would probably be more readable with a `get` instead of `is`